.github/workflows/msbuild.yml

```diff
@@ -45,7 +45,10 @@ jobs:
         ref: v${{ steps.regex-match.outputs.group1 }}
     - name: Build Dafny with local Boogie
       working-directory: dafny
-      run: dotnet build Source/Dafny.sln
+      run: |
+        dotnet build Source/Dafny.sln
+        dotnet build -f net462 Source/DafnyRuntime/DafnyRuntime.csproj
```

Comment:
Are these useful frameworks to target? Who would consume a netstandard2.0 or a net462 library?

Reply:
This was partly informed by [the dependencies here](https://github.com/aws/aws-encryption-sdk-dafny/blob/mainline/aws-encryption-sdk-net/Source/AWSEncryptionSDK.csproj#L5) with the adjustment that the current runtime doesn't build with `net452` but does go work as far as `netstandard2.0`. I also took guidance from [here](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#net-standard-not-deprecated).

Reply:
If Standard allows sharing code with Framework and Core, then why would you build for Standard AND something else? 

From the linked docs:
> Most widely used libraries will multi-target for both .NET Standard 2.0 and .NET 5+. Supporting .NET Standard 2.0 gives you the most reach, while supporting .NET 5+ ensures you can leverage the latest platform features for customers that are already on .NET 5+.

Reply:
Hmm. Re-reading that page, I think you may be right. It seems as though if we built for `netstandard2.1` we might also need a separate build to support Framework (e.g., `net462`), but if we build for `netstandard2.0` we don't need that?

Reply:
Yes, looks like it. But pff, that 2.1 vs 2.0 matters so much is confusing!